GitUI/AutoCompletion/CommitAutoCompleteProvider.cs

```diff
@@ -73,7 +73,6 @@ public async Task<IEnumerable<AutoCompleteWord>> GetAutoCompleteWordsAsync(Cance
             return autoCompleteWords.Select(w => new AutoCompleteWord(w));
         }
 
-        [NotNull]
```

Comment:
When we're done with NRT annotations, we won't need `[NotNull]`, `[CanBeNull]` or `[ContractAnnotation]` any more.